### PR TITLE
additions

### PR DESCRIPTION
**Pull request by huntertj**

### CODE REVIEW

File: proxy [hunk 1]
```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```
Comment (bitwes) [inline proxy:1]: These lines have all been added to /local/ge_post_local

File: kitchen [hunk 2]
```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```
Comment (bitwes) [inline kitchen:1]: Some of this is might be in /local/ge_post_local

File: gitprompt [hunk 3]
```diff
@@ -0,0 +1,37 @@
+COLOR_RED="\033[0;31m"
```
Comment (bitwes) [inline gitprompt:1]: Think this file is ok.

**Comment by bitwes** [comment]
Some of this is already included.

**Comment by bitwes** [comment]
over a year ago you made this pull request.  I have no idea why, but i just saw it.  How ya been Tim?

**Comment by huntertj** [comment]
I got a good laugh at this. Good times :)

**Pull request merged**